reference/handlers.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: Handlers
+---
+The Praxis [`Request`](../requests/) and [`Response`](../responses/) classes encapsulate the gory
+details of the HTTP protocol; specifically, they parse requests and generate responses so your
+business logic can focus on the _content_ of messages, not the representation of that content.
+Handlers are the "glue" that Praxis uses to parse request payloads and generate response bodies
+so that your application sees requests as structured data (hashes and arrays), and responds with
+the same.
+
+Each handler understands a particular encoding mechanism: JSON, WWW-form, and so forth.
+Praxis uses a heuristic to decide which handler is appropriate for a given HTTP body.
+
+## Registration
+
+Register new handlers at application startup time by invoking the `handler` DSL method inside
+your app's config block. Each handler is identified by a short string name and implemented
+using a Module or Class that responds to a few handler-interface methods.
```

Comment:
From the docs below, it looks like it has to be a class that is initialized, not just a module?